microsoft-365/enterprise/modern-desktop-deployment-and-management-lab.md

```diff
@@ -89,20 +74,30 @@ Note: The Windows 11 version of the lab includes the Windows 10 lab guide. A Win
 - Microsoft 365 Apps for enterprise Deployment on Non-AD Joined Devices
 - Enterprise managed deployment using Configuration Manager
 - Enterprise managed deployment using Microsoft Intune
+- Servicing Microsoft 365 Apps for enterprise using Configuration Manager
+- Servicing Microsoft 365 Apps for enterprise using Imtune
 - LOB Deployment and Management with Microsoft Intune
 - Deploy Microsoft Teams
-- NEW! Assignment filters
+- Assignment filters
+
+### Managing Microsoft Edge 
+
+- Deploy and Update Edge	
+- IE Mode
+- Setup Enterprise New Tab Page	
 
-### Deploy Windows Virtual Desktop
+### Security and Compliance 
 
-- Prepare, deploy, optimize
+- BitLocker	
+- Windows Defender Antivirus	
```

Comment:
I'm not the expert, but think this brand is now "_Microsoft_ Defender Antivirus" - but double-check that first. For example, https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/microsoft-defender-antivirus-windows